users/tests.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from .models import Profile
+from django.contrib.auth.models import User
+
+
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+
+
+@pytest.fixture
+def new_user():
+    return User(username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, password=PASSWORD, email=EMAIL)
+
+
+@pytest.mark.django_db()
+def test_persist_user(new_user):
+    new_user.save()
+    assert new_user in User.objects.all()
+    assert new_user.profile in Profile.objects.all()
+
+
+@pytest.mark.django_db()
+def test_new_user(new_user):
+    assert new_user.username == USERNAME
+    assert new_user.first_name == FIRSTNAME
+    assert new_user.last_name == LASTNAME
+    assert new_user.password == PASSWORD
+    assert new_user.email == EMAIL
```

Comment:
Why do you need to check this?

Reply:
To validate that our user values are the same as the user inputs